test/e2e/network/networking_perf.go

```diff
@@ -160,3 +172,227 @@ var _ = SIGDescribe("Networking IPerf [Experimental] [Slow] [Feature:Networking-
 		}
 	})
 })
+
```

Comment:
@mattfenwick can you add one of those 'summary' comments like you did in the other PR of how this test works, and also write "TODO remove other tests", so that once this is stable we can throw away the old tests which are mostly unreadable ?